templates/ftp_listener/src/ftp_listener/Module.md

```diff
@@ -0,0 +1,17 @@
+# FTP Listener
+
+In this template we will implement an FTP listener which will listen to a remote directory and periodically notify the 
+addition and deletion of files that comply with the specified file pattern.
+
+## How to run the template
+
+1. Alter the config file `src/ftp_listener/resources/ballerina.conf` according to your requirement. 
+
+2. Execute the following command in the project directory `ftp_listener` to run the service.
+```bash
+ballerina run ftp_listener
```

Comment:
Add the path to the conf file as well.
--b7a.config.file=path/to/file

Reply:
Added.